core/src/main/java/io/questdb/ServerMain.java

```diff
@@ -494,6 +497,20 @@ protected String webConsoleSchema() {
         return "http";
     }
 
+    public static final class AsyncMunmapJob extends SynchronizedJob {
```

Comment:
nit: why don't we make this top-level class? This location feels a little random